lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -307,44 +298,20 @@ class PolicyElementAssociation < ::ActiveRecord::Base
                                       columns: [:user_attribute_id, :object_attribute_id, :operation_set_id]
                                     }
 
-
-      # requires a join table (should be indexed!)
-      has_and_belongs_to_many :operations, class_name: "PolicyMachineStorageAdapter::ActiveRecord::Operation", join_table: 'operations_policy_element_associations'
-
       belongs_to :user_attribute
       belongs_to :object_attribute
       belongs_to :operation_set
 
-      #TODO: ActiveRecord's generated operations= method is inefficient, makes 1 query for each op added or removed even though there's no hooks
-      # Awkward manual implementation for now, but in the future change this to an hstore or something in the postgres adapter,
-      # and/or fix Rails.
-      def operations=(updated_operations_objects)
-        updated_set_of_operation_objects = Set.new(updated_operations_objects)
-        current_set_of_operation_objects = Set.new(self.operations)
-        new_operation_objects = updated_set_of_operation_objects - current_set_of_operation_objects
-        removed_operation_objects = current_set_of_operation_objects - updated_set_of_operation_objects
-        transaction do
-          OperationsPolicyElementAssociation.where(policy_element_association_id: self.id)
-                                            .where(operation_id: removed_operation_objects.map(&:id))
-                                            .delete_all
-          OperationsPolicyElementAssociation.import([:policy_element_association_id, :operation_id],
-                                                     new_operation_objects.map { |op| [self.id, op.id] },
-                                                     validate: false)
-        end
-        self.clear_association_cache
-      end
-
-      def self.add_association(user_attribute, set_of_operation_objects, operation_set, object_attribute, policy_machine_uuid)
+      def self.add_association(user_attribute, operation_set, object_attribute, policy_machine_uuid)
```

Comment:
i think long run we were going to split out association/assignment tables entirely per pm